Containerfile

```diff
@@ -44,6 +44,9 @@ COPY kmods-sys_files /tmp/kmods-files
 
 COPY --from=ghcr.io/ublue-os/akmods:main-${FEDORA_MAJOR_VERSION} /rpms /tmp/akmods-rpms
 
+# Workaround for podman issue upstream.
+RUN rpm-ostree override replace https://bodhi.fedoraproject.org/updates/FEDORA-2023-00c78aad58
```

Comment:
Maybe a stupid question, but will this install the F39 version of Podman on F38 images too?

Reply:
I think we'd want to pin it on 38 too, though I suspect they'll fix the issue upstream before upgrading 38 anyway.